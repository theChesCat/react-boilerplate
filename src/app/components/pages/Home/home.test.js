import React from 'react'
import { shallow } from 'enzyme'

import HomeComponent from './home.component.js'

jest.mock('react-redux', () => {
    const ActualReactRedux = require.requireActual('react-redux')
    return {
        ...ActualReactRedux,
        useSelector: jest.fn().mockImplementation(() => {
            return { data: 'hello world' }
        }),
        useDispatch: jest.fn().mockImplementation(() => {
            return {}
        }),
    }
})

describe('HomeComponent', () => {
    it('should render correctly', () => {
        const component = shallow(<HomeComponent />)
        expect(component).toMatchSnapshot()
    })
})
