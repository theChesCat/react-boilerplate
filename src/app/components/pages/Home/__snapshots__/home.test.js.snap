// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should render correctly 1`] = `
<styled.div>
  <Component />
</styled.div>
`;
